engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/client/CachedConfluentSchemaRegistryClient.scala

```diff
@@ -0,0 +1,42 @@
+package pl.touk.nussknacker.engine.avro.schemaregistry.confluent.client
+
+import cats.data.Validated
+import com.typesafe.scalalogging.LazyLogging
+import io.confluent.kafka.schemaregistry.client.{SchemaMetadata, SchemaRegistryClient => CSchemaRegistryClient}
+import org.apache.avro.Schema
+import pl.touk.nussknacker.engine.avro.AvroUtils
+import pl.touk.nussknacker.engine.avro.schemaregistry.SchemaRegistryError
+
+import scala.collection.concurrent.TrieMap
+
+class CachedConfluentSchemaRegistryClient(val client: CSchemaRegistryClient) extends ConfluentSchemaRegistryClient with LazyLogging {
+
+  override def getLatestSchema(subject: String): Validated[SchemaRegistryError, Schema] = {
```

Comment:
Can you write a comment that fetching latest schema is not cached (for now?), only parsing part is cached.

Reply:
Done.